client/app/components/onboardingGuide/GuideCoverPhotoPage.jsx

```diff
@@ -1,47 +1,41 @@
-import React, { PropTypes } from 'react';
+import { PropTypes } from 'react';
+import r, { div, h2, p, img, a } from 'r-dom';
 import css from './styles.scss';
 
 import GuideBackToTodoLink from './GuideBackToTodoLink';
 
-const GuideCoverPhotoPage = (props) => (
-  <div className="container">
-    <GuideBackToTodoLink
-      changePage={props.changePage}
-      initialPath={props.initialPath}
-      t={props.t}
-    />
-
-    <h2 className={css.title} >{props.t('title')}</h2>
-
-    <p className={css.description} >
-      {props.t('description')}
-    </p>
-
-    {props.pageData.info_image ?
-      <div className={css.sloganImageContainer} >
-        <img src={props.pageData.info_image}
-          className={css.sloganImage}
-          alt={props.t('info_image_alt')}
-        />
-      </div> :
-      null}
-
-    <div className={css.infoTextContainer} >
-      <div className={css.infoTextIcon}
-        dangerouslySetInnerHTML={{ __html: props.infoIcon }} // eslint-disable-line react/no-danger
-      ></div>
-      <div className={css.infoTextContent}
-        dangerouslySetInnerHTML={{ __html: props.t('advice') }} // eslint-disable-line react/no-danger
-      ></div>
-    </div>
-
-    <a href={props.pageData.link}
-      className={css.nextButton}
-    >
-      {props.t('add_your_own')}
-    </a>
-  </div>
-);
+const GuideCoverPhotoPage = (props) => {
+  const { changePage, initialPath, t, pageData, infoIcon } = props;
+
+  return div({ className: 'container' }, [
+    r(GuideBackToTodoLink, { changePage, initialPath, t }),
+    h2({ className: css.title }, t('title')),
+    p({ className: css.description }, t('description')),
+
+    pageData.info_image ?
+      div({ className: css.sloganImageContainer }, [
+        img({
+          className: css.sloganImage,
+          src: pageData.info_image,
+          alt: t('info_image_alt'),
+        }),
+      ]) :
+    null,
```

Comment:
I guess, this should be indented more.
('if's become quite hard to follow with this style)